lms/envs/devstack.py

```diff
@@ -497,6 +497,13 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS = 'edx.devstack.kafka:29092'
 EVENT_BUS_TOPIC_PREFIX = 'dev'
 
+######################## Subscriptions API SETTINGS ########################
+SUBSCRIPTIONS_ROOT_URL = "http://host.docker.internal:18750"
```

Comment:
This looks suspicious, should this be `localhost` or `edx.devstack.{service_name}`?